chandra_models/xija/mups_valve/pm1thv2t_spec.json

```diff
@@ -125,6 +125,15 @@
     "datestop": "2019:181:23:54:30.816",
     "dt": 328.0,
     "evolve_method": 1,
+    "limits": {
+        "pm1thv2t": {
+            "planning.warning.high": 210,
+            "planning.warning.high.matlab_identifier": "yellow_limit",
```

Comment:
The MUPS guideline says "Do not exceed 210 deg F for the predicted MUPS thruster valve temperatures ( PM1THV2T, or PM2THV1T) as determined by the MCC thermal model predictions". So 210 is the official do-not-exceed planning limit => `planning.warning.high`. That planning limit is apparently `yellow_limit` in MATLAB.

The 240 limit is the ODB hardware limit. (I'm not 100% sure it is in the ODB, but that value is the don't-ever-exceed-or-it-might explode limit. In our nomenclature that would be `odb.warning.high`. 

I've been a little liberal with `odb`, but maybe we should call that instead `hardware` to imply a real hardware limit regardless of where it might be captured.

Reply:
I like the `hardware` & `planning` naming convention (vs. `odb` & `planning`).  I think it is a bit more intuitive/explicit.